code/modules/mob/living/basic/turtle/turtle.dm

```diff
@@ -0,0 +1,240 @@
+#define PATH_PEST_KILLER "path_pest_killer"
+#define PATH_PLANT_HEALER "path_plant_healer"
+#define PATH_PLANT_MUTATOR "path_plant_mutator"
+#define REQUIRED_TREE_GROWTH 250
+#define UPPER_BOUND_VOLUME 50
+#define LOWER_BOUND_VOLUME 10
+
+/mob/living/basic/turtle
+	name = "turtle"
+	desc = "Dog."
+	icon_state = "turtle"
+	icon_living = "turtle"
+	icon_dead = "turtle_dead"
+	base_icon_state = "turtle"
+	icon = 'icons/mob/simple/pets.dmi'
+	butcher_results = list(/obj/item/food/meat/slab = 3, /obj/item/food/pickle = 1, /obj/item/stack/sheet/mineral/wood = 10)
+	mob_biotypes = MOB_ORGANIC | MOB_PLANT
+	mobility_flags = MOBILITY_FLAGS_REST_CAPABLE_DEFAULT
+	health = 100
+	maxHealth = 100
+	speed = 5
+	verb_say = "snaps"
+	verb_ask = "snaps curiously"
+	verb_exclaim = "snaps loudly"
+	verb_yell = "snaps loudly"
+	faction = list(FACTION_NEUTRAL)
+	ai_controller = /datum/ai_controller/basic_controller/turtle
+	///our displayed tree
+	var/mutable_appearance/grown_tree
+	///growth progress of our tree
+	var/list/path_growth_progress = list(
+		PATH_PLANT_HEALER = 0,
+		PATH_PLANT_MUTATOR = 0,
+		PATH_PEST_KILLER = 0,
+	)
+	///what nutrients leads to each evolution path
+	var/static/list/path_requirements = list(
+		//plant healers
+		/datum/reagent/plantnutriment/eznutriment = PATH_PLANT_HEALER,
+		/datum/reagent/plantnutriment/robustharvestnutriment = PATH_PLANT_HEALER,
+		/datum/reagent/plantnutriment/endurogrow = PATH_PLANT_HEALER,
+		//plant mutators
+		/datum/reagent/plantnutriment/left4zednutriment = PATH_PLANT_MUTATOR,
+		/datum/reagent/uranium = PATH_PLANT_MUTATOR,
+		//pest killers
+		/datum/reagent/toxin/pestkiller = PATH_PEST_KILLER,
+	)
+	///if we are fully grown, what is our path
+	var/developed_path
+	///our last east/west direction
+	var/last_direction = WEST
+
+/mob/living/basic/turtle/Initialize(mapload)
+	. = ..()
+
+	desc = pick(
+		"Likely Dog...",
+		"Praise the Dog!",
+		"Dog ahead.",
+		"Could this be a Dog?",
```

Comment:
These are nice, however the full description should be a smidge more descriptive, hinting that trees can grow on these if possible.

Reply:
theres alot more details in /examine() .  if u feel anything's missing from there i can add it

Reply:
Oh, I missed that, good then.